src/stories/components/footer/description-footer.tsx

```diff
@@ -13,23 +13,18 @@ interface Props {
 
 const DescriptionFooter = ({ title, children, style = {} }: Props) => {
   const { isLight } = useThemeContext();
-  const router = useRouter();
 
-  const HandleOnClick = useCallback(
-    (url: string) => () => {
-      router.push(url);
-    },
-    [router]
-  );
   return (
     <div style={style}>
       <StyleTitle isLight={isLight}>{title}</StyleTitle>
       {children &&
         children.map((item, index) =>
           item.isNotLink ? (
-            <CookiesLink isLight={isLight} onClick={HandleOnClick(item.url)} key={index}>
-              {item.title}
-            </CookiesLink>
+            <Link href={item.url} passHref legacyBehavior>
+              <CookiesLink isLight={isLight} key={index}>
```

Comment:
You should move up the key to avoid issues

Reply:
ok

Reply:
You are right